### DISCUSSION TITLE
Strange Happenings

### DISCUSSION THREAD

**Discussion by Tim-K5AC** [aqualinkd/AqualinkD#355] [General]
Last night we were awakened by our pool lights going on/off at a rapid rate at 3:00am.  I went to the Raspberry Pi-4/Touchscreen and noticed that the AqualinkD browser was showing the pool light going on/off and it showed the Spa heater was enabled (the pump is off).  I didn't see any unusual messages on the serial bus.   The lights are ONLY controlled by AqualinkD (ie no schedules in the RS panel).   I shut down the AqualinkD service and rebooted the Pi.  When I tried to restart AqualinkD it said that the file aqualinkd.conf had a bad file descriptor.  I edited the file and saved it to a new filename and got the same error message with the backup file.

I did a clean install of aqualinkD and installed the release and then copied my aqualinkd.conf file and everything started up correctly.

Has anyone had this issue before?
When I did the clean, did it also wipe out my chron schedules (where are they stored)?

Tim P

**Comment by sfeakes** [comment]
Schedule is in /etc/cron.d/aqualinkd

Sounds like you had some data corruption, if you are using CF card you may want to replace it.

as for light (or device) turning on / off, I have only seen something similar one time and that was when someone was using a wifi RS485 adapter. When the wifi network introduced latency the jandy panel got confused with packet order.  If you are using wired adapter it’s not that.  I tent more towards OS file system stability from your description.

You should be able to still pull the aqualinkd logs from that time using journalcrl as long as they are not corrupt.  Wiki has info in the log section

**Comment by bcre3306** [comment]
I can attest to what is being said above, I was the one with the EW11 which was the RS485 wifi adapter and it indeed caused the random on/off. Once I swapped to a raspberry it worked flawlessly.
